src/pages/layer-2.js

```diff
@@ -0,0 +1,1096 @@
+    flex-direction: column;
+  }
+`
+
+const StatBox = styled.div`
+  display: flex;
+  flex-direction: column;
+  flex: 33%;
+  padding: 0 20px;
+  text-align: center;
+  align-content: center;
+  justify-content: center;
+  @media (max-width: ${({ theme }) => theme.breakpoints.m}) {
+    flex: 100%;
+  }
+`
+
+const StatPrimary = styled.p`
+  font-weight: bold;
+  font-size: ${(props) => props.theme.fontSizes.xl};
+  color: ${(props) => props.theme.colors.primary};
+  font-family: monospace;
+`
+
+const StatDescription = styled.p`
+  opacity: 0.8;
+  margin: 0;
+`
+
+const StatDivider = styled.div`
+  border-left: 1px solid ${({ theme }) => theme.colors.homeDivider};
+  max-height: 100px;
+  @media (max-width: ${({ theme }) => theme.breakpoints.m}) {
+    border-left: none;
+    border-bottom: 1px solid ${({ theme }) => theme.colors.homeDivider};
+    width: 100%;
+    height: 0%;
+    margin: 2rem 0;
+  }
+`
+
+const Layer2Page = ({ data }) => {
+  const intl = useIntl()
+
+  const [tvl, setTVL] = useState("-")
+  const [percentChangeL2, setL2PercentChange] = useState("-")
+  const [averageFee, setAverageFee] = useState("-")
+
+  useEffect(() => {
+    const fetchL2Beat = async () => {
+      try {
+        const l2BeatData = await getData("https://l2beat.com/api/tvl.json")
+        // formatted TVL from L2beat API formatted
+        setTVL(
+          new Intl.NumberFormat(intl.locale, {
+            style: "currency",
+            currency: "USD",
+            notation: "compact",
+            minimumSignificantDigits: 2,
+            maximumSignificantDigits: 3,
+          }).format(l2BeatData.data[l2BeatData.data.length - 1][1])
+        )
+        // Calculate percent change ((new value - old value) / old value) *100)
+        setL2PercentChange(
+          (
+            ((l2BeatData.data[l2BeatData.data.length - 1][1] -
+              l2BeatData.data[l2BeatData.data.length - 31][1]) /
+              l2BeatData.data[l2BeatData.data.length - 31][1]) *
+            100
+          ).toFixed(2)
+        )
+      } catch (error) {
+        console.error(error)
+      }
+    }
+
+    fetchL2Beat()
+
+    const fetchCryptoStats = async () => {
+      try {
+        // Average eth transfer fee from L2's supported by cryptostats API
+        const feeData = await getData(
+          "https://api.cryptostats.community/api/v1/l2-fees/feeTransferEth?metadata=false"
+        )
+        const feeAverage =
+          feeData.data.reduce(
+            (acc, curr) => (acc += curr.results.feeTransferEth),
+            0
+          ) / feeData.data.length
+        setAverageFee(feeAverage.toFixed(2))
+      } catch (error) {
+        console.error(error)
+      }
+    }
+    fetchCryptoStats()
+  }, [])
+
+  const heroContent = {
+    title: "Layer 2",
+    header: "Layer 2",
+    subtitle:
+      "Scaling Ethereum without compromising on security and decentralization.",
+    image: getImage(data.ethBlocks),
```

Comment:
I wonder if this rollup image would be a better hero here? Just my 2 gwei but it feels more descriptive to me:

![image](https://user-images.githubusercontent.com/8097623/160943098-1f2cd27d-7ce0-4bb7-bd6c-0c33e43ed25e.png)

Reply:
Agreed. I see Corwin has already made the change. Are we okay with using this twice on the page?

Not against, just posing the question. (I think it's fine)

Reply:
I was thinking about this and made a different image to the hero, this one is to close to rollups
![l2hero](https://user-images.githubusercontent.com/1120748/161059991-765a1a9f-6647-4db4-a0df-f4b6ee97ceb2.png)
This is my take for the her
![Screen Shot 2022-03-31 11 15 53 AM](https://user-images.githubusercontent.com/1120748/161060041-2240f7a9-9e7e-4c4f-8bfe-645aa66434f7.png)
o